src/components/Modals/Send/views/Status.tsx

```diff
@@ -0,0 +1,100 @@
+import { CheckCircleIcon, WarningIcon } from '@chakra-ui/icons'
+import { Card } from '@chakra-ui/react'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import React, { useCallback, useMemo } from 'react'
+import { useWatch } from 'react-hook-form'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+import { useModal } from 'hooks/useModal/useModal'
+import { useTxStatus } from 'hooks/useTxStatus/useTxStatus'
+import { getTxLink } from 'lib/getTxLink'
+import { SharedStatus } from 'pages/RFOX/components/Shared/SharedStatus'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { SendInput } from '../Form'
+import { SendFormFields } from '../SendCommon'
+
+type BodyContent = {
+  key: TxStatus
+  title: string
+  body: string | [string, Record<string, string | number>]
+  element: JSX.Element
+}
+
+export const Status: React.FC = () => {
+  const send = useModal('send')
+  const qrCode = useModal('qrCode')
+
+  const handleClose = useCallback(() => {
+    send.close()
+    qrCode.close()
```

Comment:
https://github.com/shapeshift/web/commit/00ad0c2afe6e6765f1d474953e3883b1fea00664#diff-71a414ff9cc9b6ed2cd88ee5c6a51ba0e10d2b68d671ceb527afc50eee124dddR91-R92

I think there is a world where the modal object can be undefined and so its functions, I didnt test at runtime yet but pretty sure there is an error in the console everytime we close the modal

Reply:
Couldn't reproduce at runtime on both QRCode modal and basic send modal